Tutorials/intro-to-octave/set_octave.py

```diff
@@ -134,7 +134,7 @@ def octave_settings(qmm, qm, prog, config, octave_config, external_clock=False,
         # Change to the relevant external frequency
         qmm.octave_manager.set_clock("octave1", ClockType.External, ClockFrequency.MHZ_10)
         # If using a clock from the OPT, use this command instead
-        # qmm.octave_manager.set_clock(octave, ClockType.Buffered, ClockFrequency.MHZ_1000)
+        # qmm.octave_manager.set_clock("octave1", ClockType.Buffered, ClockFrequency.MHZ_1000)
```

Comment:
In this file (in the Octave configuration) the user has the possibility to add a second Octave so maybe it would be good to have the possibility to also configure the second Octave by looping over the octaves from the octave_config, what do you think?

Reply:
I intended to make it with more octaves but didn't get to it... You're right. :)